_includes/footer.html

```diff
@@ -0,0 +1,16 @@
+<footer class="footer">
+	<div class="container is-fluid">
+		<div class="footer-wrapper">
+			<div class="footer-wrapper-left">
+				<p>
+					OpenFaaS ® was founded by <a href="https://www.alexellis.io/">alexellis</a> and is being built by a growing <a href="https://kenfdev.o6s.io/github-stats-page#/">community of contributors.</a> <br>
+					OpenFaaS Ltd. is a registered company in England & Wales with number: 11076587, registered address: Peterborough UK.
+				</p>
+			</div>
+			<div class="footer-wrapper-right">
+				<small>Featured on</small>
+				<img src="{{ site.baseurl }}/images/openfaas/cloud-native.png">
```

Comment:
Maybe this should be a link to the serverless landscape? 

https://landscape.cncf.io/format=serverless